Misc/holorepo_demo/README.md

```diff
@@ -0,0 +1,77 @@
+# HoloRepository Demo
```

Comment:
The main `README` should have new section that redirects readers here w.r.t. to setting up everything